scripts/copy-latest.py

```diff
@@ -0,0 +1,34 @@
+import re
+import shutil
+import os
+from pathlib import Path
+
+pattern = re.compile("(data-version=\")((\d+\.)(\d+\.)(\*|\d+)\.Final)")
+found = False
+base_path = "build/site/serverlessworkflow"
+index_file = Path(base_path + "/latest/index.html").resolve()
+
+for i, line in enumerate(open(index_file)):
+    for match in re.finditer(pattern, line):
+        found = True
+        if len(match.groups()) > 2:
+            latest_version = match.groups()[1]
+            print("Found latest version " +
+                  latest_version + ". Copying directory.")
+            dest_dir = Path(base_path + "/" + latest_version)
+            if not dest_dir.is_dir():
+                shutil.copytree(Path(base_path + "/latest").resolve(),
+                                dest_dir.resolve())
+            else:
+                print("Directory with version " + latest_version +
+                      " already exists. Skipping copy.")
```

Comment:
No, at this stage, the website is already built with the right versions. We shouldn't do anything. It's here just as a guard code in case something we haven't prevented happens.